js/ui/diagram/diagram.importer.js

```diff
@@ -1,12 +1,25 @@
-const Errors = require('../widget/ui.errors');
-const Diagram = require('devexpress-diagram');
+import Errors from '../widget/ui.errors';
```

Comment:
@DokaRus Please review this file.

Reply:
We're removed this logic in this [PR](https://github.com/DevExpress/DevExtreme/pull/12260). 
The main reason why `import` were not used to receive `devexpress-gantt` and `devexpress-diagram` packages did not have default exports. So the code was left with `require` as requiring the least amount of rework.